ui/package.json

```diff
@@ -142,16 +145,13 @@
     "ember-cli-terser": "^4.0.2",
     "ember-cli-typescript": "^5.2.1",
     "ember-composable-helpers": "5.0.0",
-    "ember-concurrency": "2.3.4",
-    "ember-copy": "2.0.1",
-    "ember-d3": "^0.5.1",
```

Comment:
i removed this in favor of installing `d3-array`, `d3-format`, and `d3-shape` directly, since those were the packages we were actually importing.